openedx/core/djangoapps/util/storage.py

```diff
@@ -0,0 +1,18 @@
+"""
+Django storage utilities.
+"""
+from django.core.files.storage import get_storage_class
+from django.utils.lru_cache import lru_cache
+
+
+@lru_cache()
+def get_storage(storage_class=None, **kwargs):
```

Comment:
Great points. I'm going to revert all these changes! I did it initially because there was a gnarly Python circularity, due to middleware loading these storage classes even if they weren't used. However, this is no longer an issue in my more current (uncommited) changes because I'm not handling the post updates in middleware anymore.